### PR TITLE
It's not you, it's me

### PR DESCRIPTION
**Pull request by jessieay**
* A rejection email
* Closes https://github.com/18F/micropurchase/issues/1018

screen shot 2016-08-24 at 2 39 13 pm

### CODE REVIEW

File: config/locales/mailers/en.yml [hunk 1]
```diff
@@ -60,3 +60,12 @@ en:
         para_2: "Again, great job! We look forward to working with you again in the future."
         sign_off: "Sincerely,"
         from: "The 18F Micro-purchase team"
+      auction_rejected:
+        subject: "18F Micro-purchase Auction Status"
+        para_1: >-
+          We regret to inform you that your work for %{auction_title} was rejected
```
Comment (harrisj) [inline config/locales/mailers/en.yml]: This message suggests that we will provide a reason for rejection on the auction. Is that true?

**Comment by harrisj** [comment]
So, this is an interesting thing, because while I am generally for automating things, I realize there are several reasons why we might reject an auction. What's weird is that in each of these cases, we would probably relay the reason _why_ we rejected the auction in a different way.

* No work submitted (this would just be in the email sent, check if auction rejected and no delivery_url)
* Work was submitted but substandard (here we would probably comment on the delivery_url PR)
* We can't actually pay the person because they are an excluded entity or such (I guess this would mean a special email)

Am I missing any other circumstances where a person would be rejected?

**Comment by jessieay** [comment]
great points, @harrisj ! Looking to @mtorres253 for some direction on this issue...

**Comment by mtorres253** [comment]
@jessieay @harrisj Yes I was just thinking about this today. It would be nice to add a comment or reason for the rejection. We can groom this on Monday and add some stories. For now we can handle manually (if necessary)

**Comment by jessieay** [comment]
ok so should we not move forward with this? close #1018 without merging?

**Comment by mtorres253** [comment]
@jessieay @harrisj I think we should merge this. At least they can get an email notification that the auction has been rejected. We don't need to systematize reasons for rejection at this point. It will probably be clear in the github issue why the deliverable was rejected.

**Comment by jessieay** [comment]
ok @harrisj I will leave it up to you

**Comment by adelevie** [comment]
I agree with @mtorres253. But @harrisj brings up a great point. Maybe we also say, feel free to email us back/get in touch and that we're always happy to discuss what didn't go well and what could be better. Also, maybe include a standing offer to retro???

Makes me also think: could/should our emails always include micropurchase@gsa.gov so that they become reply-able?

**Comment by jessieay** [comment]
We have to send emails from an 18F domain (18f.gov) because of the way
mandrill works.

We do have micropurchsse@gsa as the reply to address on all emails, though

**Comment by adelevie** [comment]
Couldn't we cc the @gsa.gov account? Or add it to the `to:` field? That way that account will also catch replies?

**Comment by jessieay** [comment]
when you set the `reply-to` that is the address that is used for replies. Is there a different scenario you're thinking of that I am missing?

**Comment by jessieay** [comment]
here is a very retro but informative overview of `reply-to` vs `from` http://emailuniverse.com/ezine-tips/?id=179

**Comment by jessieay** [comment]
will wait to hear back from @mtorres253 on whether we should tweak copy before merging.

File: config/locales/mailers/en.yml [hunk 2]
```diff
@@ -60,3 +60,12 @@ en:
...
+          on %{rejected_at}. For more information please see the auction details here
+          %{auction_url} or contact us at micropurchase@gsa.gov. Though this auction
```
Comment (mtorres253) [inline config/locales/mailers/en.yml]: For more information please see GitHub issue: %{link to GitHub issue}. Please also let us know if you would like us to schedule a retrospective to go over any questions or concerns you may have.

**Comment by adelevie** [comment]
🍧

**Pull request merged**